backvendor/gosource.go

```diff
@@ -93,6 +102,38 @@ func NewGoSource(pth string, excludeGlobs ...string) (*GoSource, error) {
 	return src, nil
 }
 
+// readGodepsConf parses Godeps/Godeps.json to extract the package
+// name.
+func readGodepsConf(src *GoSource) error {
```

Comment:
I'm not sure if this is usual in Go, but when I read a function called `readGodepsConf`, I don't assume it's going to change the value of the struct I passed in. Perhaps a better name for this function is `loadGodepsConf`.

Reply:
Good idea, renamed.